### ISSUE TITLE
make -j2 not working

### ISSUE THREAD

**Issue by Lecrapouille** [philburk/pforth#52]
Cannot compile pForth with multiple cores (make -jX for example -j2 -j4 -j8) on Linux (ie Debian 9.3) Got the (translated) failure:
```
cd pforth/build/unix
make -j2

../../csrc/pf_save.c:42:26: fatal error: pfdicdat.h: no such file or directory
compilation terminated.
Makefile:80 : target « pf_save.eo » failed
```
while `make` alone works and finish compiling. I think its just a story of order prerequisites and just a `|` is missing (but I failed to place it correctly). See http://www.gnu.org/software/make/manual/make.html#Prerequisite-Types

**Comment by letoh** [comment]
The latest commit looks fine, I think this issue could be closed.

**Comment by Lecrapouille** [comment]
@letoh the fix has never been merged

**Comment by letoh** [comment]
> 
> 
> @letoh the fix has never been merged

@Lecrapouille I know the workaround is not merged, but we don't need it anymore. PR #80 should solve this issue.

Here is some test results:
| commit | make version | patch | result | |
| - | - | - | - | - |
| ee8dc9e | 3.8x |   | error |
| ee8dc9e | 3.8x | #80 | pass |
| ee8dc9e | 4.3 |  | error | see #76 |
| ee8dc9e | 4.3 | #80 | pass |
| e476e08 | 3.8x | | error |
| e476e08 | 3.8x | #80 | pass |
| e476e08 | 4.3 | | error |
| e476e08 | 4.3 | #80 | pass |

**Comment by philburk** [comment]
Fixed by #80

**Comment by Lecrapouille** [comment]
Worked fine for me with -j8 on an up to date Debian 10.
